tsl/src/compression/gorilla.c

```diff
@@ -254,9 +254,11 @@ gorilla_compressor_alloc(void)
 	GorillaCompressor *compressor = palloc(sizeof(*compressor));
 	simple8brle_compressor_init(&compressor->tag0s);
 	simple8brle_compressor_init(&compressor->tag1s);
-	bit_array_init(&compressor->leading_zeros);
+	bit_array_init(&compressor->leading_zeros,
+				   /* expected_bits = */ GLOBAL_MAX_ROWS_PER_COMPRESSION * 1);
 	simple8brle_compressor_init(&compressor->bits_used_per_xor);
-	bit_array_init(&compressor->xors);
+	bit_array_init(&compressor->xors,
+				   /* expected_bits = */ GLOBAL_MAX_ROWS_PER_COMPRESSION * 12);
```

Comment:
Wny `12` here?

Reply:
Typical value we observe for some data sets. I'll add a comment.